modules/xsession.nix

```
@@ -33,8 +33,15 @@ in {
           in
             "''${xmonad}/bin/xmonad";
         '';
+        default = ''test -n "$1" && eval "$@"'';
         description = ''
-          Window manager start command.
+          Window manager start command. The default value handles
+          display-manager parameter from <code>de+wm.desktop</code>. If you use
+          home-manager to config windown manager on NixOS, you have to handle
```

typo, and I'm also not quite sure what you are trying to get across with that sentence. Maybe this?

```suggestion
          home-manager to config the window manager on NixOS, you have to handle
```

---

Sorry. It is fixed now.

---

Sorry for another nit, but maybe it should be "your" instead of "the"?

```
home-manager to config your window manager...
```